src/web/components/App.tsx

```diff
@@ -174,12 +215,14 @@ export const App = ({ CAPI, NAV }: Props) => {
         // Note: Both require a 'root' element that needs to be server rendered.
         <React.StrictMode>
             <Portal root="reader-revenue-links-header">
-                <ReaderRevenueLinks
-                    urls={CAPI.nav.readerRevenueLinks.footer}
-                    edition={CAPI.editionId}
-                    dataLinkNamePrefix="footer : "
-                    inHeader={true}
-                />
+                <Suspense fallback={<></>}>
+                    <ReaderRevenueLinks
+                        urls={CAPI.nav.readerRevenueLinks.footer}
+                        edition={CAPI.editionId}
+                        dataLinkNamePrefix="footer : "
+                        inHeader={true}
+                    />
+                </Suspense>
```

Comment:
This component is in the header so there's no benefit to lazy loading it?

Reply:
if the user scrolls down the page fast, or the page is small and both are trying to be rendered, the fact this component is not wrapped in a `Suspense`, will cause the JS to crash. 

The component will on be fetched once anyway, `Suspense` is just a wrapper to handle the state while the component is loading

Reply:
I think though that, if the component isn't being lazy loaded, then we shouldn't dynamically load it, no?

Reply:
oh I see what you mean, yeah good point, ill remove the dynamic import